py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -1170,68 +1175,112 @@ def convert_exported_program_to_serialized_trt_engine(
             )
 
     if (
-        not immutable_weights
-        and not refit_identical_engine_weights
-        and enable_weight_streaming
+        "enable_cross_compile_for_windows" in kwargs.keys()
+        and kwargs["enable_cross_compile_for_windows"]
     ):
         raise ValueError(
-            "TensorRT's `REFIT` flag is not compatible with `enable_weight_streaming=True` for now. This issue was reported on https://github.com/pytorch/TensorRT/issues/3305"
+            "Please use cross_compile_for_windows() api if you want to cross compile the module in Linux for inferencing in Windows."
```

Comment:
Can this be rephrased to `Please use torch_tensorrt.dynamo.cross_compile_for_windows() if you want to cross compile the module in Linux for inferencing in Windows.`